lib/config.js

```diff
@@ -13,8 +13,9 @@ function Config (process) {
   this.httpProxy = process.env.IED_HTTP_PROXY || process.env.HTTP_PROXY || null
   this.httpsProxy = process.env.IED_HTTPS_PROXY || process.env.HTTPS_PROXY || null
   this.requestRetries = parseInt(process.env.IED_REQUEST_RETRIES, 10) || 5
-  this.sh = process.env.IED_SH || (process.platform === 'win32' ? process.env.comspec || 'cmd' : 'sh')
-  this.shFlag = process.env.IED_SH_FLAG || (process.platform === 'win32' ? '/d /s /c' : '-c')
+  this.sh = process.env.IED_SH || (isWindows ? process.env.comspec || 'cmd' : 'sh')
+  this.shFlag = process.env.IED_SH_FLAG || (isWindows ? '/d /s /c' : '-c')
```

Comment:
`/d`: Disables execution of AutoRun commands.
`/s`: Modifies the treatment of string after /c or /k.
`/c`: Carries out the command specified by string and then stops.

From https://www.microsoft.com/resources/documentation/windows/xp/all/proddocs/en-us/cmd.mspx?mfr=true

So I think `/c` is the same behavious as `-c` in `sh`, `/s` strips opening and closing quotation marks and `/d` disables commands specified in the registry to always run when `cmd.exe` is executed.